src/views/accounts/Automation/BaseExecutionList.vue

```diff
@@ -168,6 +158,15 @@ export default {
     if (automation_id !== undefined) {
       this.tableConfig.url = `${this.tableConfig.url}?automation_id=${automation_id}`
     }
+
+    const defaultExtraActions = this.tableConfig.columnsMeta.actions.formatterArgs.extraActions
+
+    if (this.customActions) {
+      this.tableConfig.columnsMeta.actions.formatterArgs.extraActions = [
+        ...defaultExtraActions,
+        this.customActions
+      ]
+    }
   },
   methods: {
     handleDetailCallback(row) {
```

Comment:
It seems that there is no syntax error detected. However, I noticed some formatting changes:
- The first line should have two spaces after the `export` keyword (`export default`).
- In the comments before each line, please include a blank line at the bottom so that they look more organized.

Here's how it looks with these improvements:

```
{
  resources: {
    type: String;
    required: true;
  },
  customActions: {
    type:Object;
  }
}
```